### ISSUE TITLE
LP#1409717 "MenuLibre does not see new launcher after reopen for non-executable in xubuntu"

### ISSUE THREAD

**Issue by bluesabre** [bluesabre/menulibre#16]
Reported first on Launchpad at https://bugs.launchpad.net/menulibre/+bug/1409717

[LP#0](https://bugs.launchpad.net/menulibre/+bug/1409717/comments/0): *GeoMint (geomint) wrote on 2015-01-12:*

> I create a new launcher with icon, name and command filed.
> I can see the new launcher in xfce4-whiskermenu-plugin (version 1.4.2)
> but when i reopen the MenuLibre it is not visible.
> I search in MenuLibre for terms but does not find my launcher.
> when i try open the launcher popups error message "Failed to execute command ... " and "Failed to execute child process  ... (Permission denied) " ( in ... says my command )

**Comment by bluesabre** [comment]
Comment: 
Also had the identical problem after installing libreoffice but after reboot, the menu item showed up in the Office submenu.  
Missing menu items are an occasional but recurring feature of menulibre as well as corruption. But, I don't know enough to track down the problem. 
The best way to deal with a corrupt menu  is to delete the menu structures in the user folders, as outlined in a post  by Sean Davies, and drag (recreate) a new XFCE menu by the plugin. 


Launchpad Details: [#LPC](https://bugs.launchpad.net/menulibre/+bug/1409717/comments/1) ineuw - 2015-01-12 15:58:30 +0000

**Comment by bluesabre** [comment]
I had the same problem creating a custom launcher but making a mistake in the name of the command.

So, MenuLibre is unable to list launchers which have executable files that don't exist or with bad permission.

The workaround in my case was simple : rename the executable (because I remembered my mistake) so my launcher was visible and then modify it before renaming the launcher again.

I checked the permission issue with a chmod a-wrx and it produces the same effect.

Launchpad Details: [#LPC](https://bugs.launchpad.net/menulibre/+bug/1409717/comments/2) Jean Claveau - 2016-03-31 09:24:23 +0000
